cmd/acceptance/main.go

```diff
@@ -144,12 +145,16 @@ func main() {
 		SetDefaultEventuallyTimeout(time.Minute)
 		SetDefaultEventuallyPollingInterval(100 * time.Millisecond)
 
-		RunSpecs(new(testing.T), "theatre/cmd/acceptance")
+		if RunSpecs(new(testing.T), "theatre/cmd/acceptance") {
+			os.Exit(0)
+		} else {
+			os.Exit(1)
+		}
 	}
 }
 
 var _ = Specify("Acceptance", func() {
```

Comment:
This feels like it should be `Describe("Consoles", ...)` to me

Reply:
Ah - so actually I disagree. This is the entrypoint for the acceptance tests for the entire theatre system. We may also put DRB tests in here, as well as anything new we add after that. I've added a Describe to the acceptance spec to mark that more clearly.